src/js/va-letters/reducers/index.js

```diff
@@ -1 +1,27 @@
-export default {};
+import set from 'lodash/fp/set';
+
+const initialState = {
+  letters: [],
+  destination: {},
+  available: false
+};
+
+function letters(state = initialState, action) {
+  switch (action.type) {
+    case 'GET_LETTERS_SUCCESS':
+      return {
+        ...state,
```

Comment:
It's object spread syntax: http://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html
It's just a little more compact than 2 `set` operations for the 2 fields that are being updated here (as compared to a single one below for `GET_LETTERS_FAILURE`